builds/e2e/e2e.yaml

```diff
@@ -254,6 +254,28 @@ jobs:
     - template: templates/e2e-clear-docker-cached-images.yaml
     - template: templates/e2e-run.yaml
 
+  ################################################################################
+  - job: redhat8_amd64
+  ################################################################################
+    displayName: RedHat8 amd64
+
+    pool:
+      name: $(pool.linux.name)
+      demands:
+      - ImageOverride -equals agent-aziotedge-rhel-8-msmoby
```

Comment:
I thought we have an issue with the disk size for the build. Is it no longer the case?

Reply:
Yes, the default host size for the home directory was only 1G, which was not enough space to do a `dotnet restore` during the build.  This is fixed on the build host by running `lvresize -r -L+5G rootvg/homelv`.